lib/systems/platforms.nix

```diff
@@ -255,6 +255,7 @@ rec {
       arch = "armv6";
       fpu = "vfp";
       float = "hard";
+      abi = "aapcs-linux";
```

Comment:
I can do that, but do be clear I also changed some of the `config` strings, so this alone won't be enough to preserve hashes. I can also keep the old config strings, but some of them now have evaluation problems because of my parsing code that needs the "unknown" OS to disambiguate.